server/validation.js

```diff
@@ -44,6 +51,11 @@ const DeterminationSchema = yup.object().noUnknown().shape({
 const PhacSchema = yup.array().required('Submission must contain at least one item').of(
   yup.object().shape({
     covid_id: yup.string().typeError('covid_id must be a string').required('covid_id is required'),
+    date_of_birth: yup.string().required('Date of birth is required').test('is-date', 'Date of birth is invalid', validatePastDateString),
```

Comment:
I think we might want to validate the structure. Asked @fwkendall to verify

```
{
        "covid_id": "CVR-test1",
        "first_name": "Test",
        "last_name": "UserMcGee",
        "date_of_birth": "1967-01-01",
        "type": "Asymptomatic",
        "arrival_date": "2020-01-01",
        "end_of_isolation": "2020-01-15",
        "destination_type": "Home",
        "address_1": "",
        "city": "",
        "postal_code": "",
        "home_phone": "555-555-5555",
        "mobile_phone": "",
        "other_phone": "",
        "email_address": "",
        "port_of_entry": "Land border",
        "province_territory": "British Columbia"
    }
```

Reply:
We probably want postal code validation. This regex worked well in the past:
`[ABCEGHJKLMNPRSTVXY]\d[ABCEGHJ-NPRSTV-Z][\s-]?\d[ABCEGHJ-NPRSTV-Z]\d`

Reply:
```
{
        "covid_id": "CVR-test1", 
        "first_name": "Test", 
        "last_name": "UserMcGee",
        "date_of_birth": "1967-01-01", // validate date
        "type": "Asymptomatic",
        "arrival_date": "2020-01-01", // required
        "end_of_isolation": "2020-01-15", // required
        "destination_type": "Home",
        "address_1": "",
        "city": "",
        "postal_code": "",
        "home_phone": "555-555-5555", // check format
        "mobile_phone": "", // check format
        "other_phone": "", // check format
        "email_address": "", // check format
        "port_of_entry": "Land border",
        "province_territory": "British Columbia" // required
    }
```

Reply:
We can validate the required fields in the above list. Others marked as `check format` need to be validated based on data from the database.